generators/app/package-json.js

```diff
@@ -143,6 +144,7 @@ const packageJSON = (current, context) => {
       'server': 'karma start scripts/karma.conf.js --singleRun=false --auto-watch --no-browsers',
       'pretest': 'npm-run-all lint build',
       'test': 'karma start scripts/karma.conf.js',
+      'posttest': 'shx cat test/dist/coverage/text.txt',
```

Comment:
If we actual report this in karma we get coverage for every browser that runs, which is almost always the same.